From 910c51dcae81d4157748cb09798acb958da312ac Mon Sep 17 00:00:00 2001
From: Austin Yuan <shengquan.yuan@gmail.com>
Date: Wed, 08 Jun 2011 01:48:19 +0000
Subject: test/basic testing programs do not need to be installed

in /usr/bin; one can run make valgrind here as a check,
or run them under make check too.

From gimli at dark-green.com

Signed-off-by: Austin Yuan <shengquan.yuan@gmail.com>
---
diff --git a/test/basic/Makefile.am b/test/basic/Makefile.am
index a2519a2..c84c264 100644
--- a/test/basic/Makefile.am
+++ b/test/basic/Makefile.am
@@ -20,7 +20,7 @@
 # TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE
 # SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 
-bin_PROGRAMS = test_01 test_02 test_03 test_04 test_05 test_06 \
+check_PROGRAMS = test_01 test_02 test_03 test_04 test_05 test_06 \
 		test_07 test_08 test_09 test_10 test_11
 
 AM_CFLAGS = -I$(top_srcdir)/va -I$(top_srcdir)/src/x11
@@ -62,7 +62,7 @@ test_11_SOURCES = test_11.c
 
 EXTRA_DIST = test_common.c
 
-valgrind:	$(bin_PROGRAMS)
-	for a in $(bin_PROGRAMS); do \
+valgrind:	$(check_PROGRAMS)
+	for a in $(check_PROGRAMS); do \
 		valgrind --leak-check=full --show-reachable=yes .libs/$$a; \
 	done
--
cgit v0.8.3-6-g21f6

